src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  const newState = state;
+
+  for (let i = 0; i < transforms.length; i++) {
```

Comment:
you can use here 'for (const obj of transforms)'  and at all you can do all the task with switch case, it's more understandable and just easy:)